external/fv3fit/fv3fit/train_microphysics.py

```diff
@@ -204,6 +208,93 @@ def __post_init__(self) -> None:
             self.cache = False
 
 
+def get_model_output_sensitivities(
```

Comment:
this file is getting rather long. Can you move the jacobian code (no plots) to a seperate module...and the plotting code elsewhere too...maybe there is an appropriate place already.

Reply:
Moved jacobian specifics into `emulation/keras.py` and the plotting into `wandb.py`.